test/helpers/remove-temp-path.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const fs = require('fs');
+const tmpPath = require('./tmp-path.js');
```

Comment:
How about having a constants file with all these test helpers one-liners? Just thinking out loud since it looks like unnecessary abstraction :)

Reply:
Yeah that'll be better.